next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   productionBrowserSourceMaps: true,
   images: {
+    qualities: [5, 25, 50, 75],
```

Comment:
When I run the app locally, I do not see a warning, but I see it in the test output. 

<img width="1414" height="492" alt="Screenshot 2025-08-31 at 5 42 36 PM" src="https://github.com/user-attachments/assets/0835aeb6-d91c-47e0-a657-22035b6c98c0" />